File: examples/H20/robots/sdk_package/example/h2x/high_level/rise.cpp
#include <iostream>

#include "h2x/h2x_high_level.h"

using namespace humanoid;
using namespace humanoid::h2x;

int main() {
  std::cout << "=== h2x 躺起示例程序 ===" << std::endl;
  H2xHighLevel high_level;

  int ret = high_level.Rise();
  if (ret == 0) {
    std::cout << "成功躺起" << std::endl;
  } else {
    std::cout << "躺起失败, 错误码：" << ret << std::endl;
    return -1;
  }
  return 0;
}

File: examples/H20/robots/sdk_package/example_py/h2x/high_level/rise.py
import humanoid_sdk_py.h2x as h2x
import time

h2x_high_level = h2x.H2xHighLevel()
time.sleep(1)

print("=== h2x 躺起示例 ===")
ret = h2x_high_level.Rise()
if ret == 0:
    print("躺起成功")
else:
    print("躺起失败，错误码：", ret)
